### DISCUSSION TITLE
Hockey Stats_map numbers?

### DISCUSSION THREAD

**Discussion by jwinik** [cwendt94/espn-api#320] [ESPN API]
As I'm reviewing this df of player stats I created, I noticed that there are columns that are just numbers that come from the stats_map dictionary. Most keys have a value pair that makes sense, but there's a handful with numeric values. Is there anywhere to see what these numbers represent?

```
STATS_MAP = {
    '0': 'GS',
    '1': 'W',
    '2': 'L',
    '3': 'SA',
    '4': 'GA',
    '5': '5',
    '6': 'SV',
    '7': 'SO',
    '8': 'MIN ?',
    '9': 'OTL',
    '10': 'GAA',
    '11': 'SV%',
    '12': '12',
    '13': 'G',
    '14': 'A',
    '15': '+/-',
    '16': '16',
    '17': 'PIM',
    '18': 'PPG',
    '19': '19',
    '20': 'SHG',
    '21': 'SHA',
    '22': 'GWG',
    '23': 'FOW',
    '24': 'FOL',
    '25': '25',
    '26': 'TTOI ?',
    '27': 'ATOI',
    '28': 'HAT',
    '29': 'SOG',
    '30': '30',
    '31': 'HIT',
    '32': 'BLK',
    '33': 'DEF',
    '34': 'GP',
    '35': '35',
    '36': '36',
    '37': '37',
    '38': 'PPP',
    '39': 'SHP',
    '40': '40',
    '41': '41',
    '42': '42',
    '43': '43',
    '44': '44',
    '45': '45',
    '99': '99'
    } 
```

**Comment by cwendt94** [comment]
The only way I have found the mapping between the numbers to stats is going on ESPN's website, looking up a players stats and then try to compare with the same data that is returned in the players stat map. I haven't found any documentation on which numbers map to which stats unfortunately.

**Reply by jwinik** [reply]
Understood. Thanks for the explanation!
